src/mobile/ios/EntangledIOS/EntangledKit.framework/Headers/Interface.h

```diff
@@ -15,13 +15,13 @@
 @interface EntangledIOSBindings : NSObject
 
 + (NSString*)iota_ios_digest:(NSString*)trytes;
-+ (NSString*)iota_ios_pow_trytes:(NSString*)trytes mwm:(NSNumber*)mwm;
++ (NSString*)iota_ios_pow_trytes:(NSString*)trytes mwm:(int)mwm;
```

Comment:
The current `Interface.h` uses `NSNumber` instead of `int`:
https://github.com/iotaledger/entangled/blob/dfc34e2b61af415c451b32078f592069a14f4590/mobile/ios/Interface.h#L18

Was this built from a different branch?